readme.md

```diff
@@ -32,37 +32,63 @@ Configuration of DOPR can be done either via `package.json` under the `dopr` key
   "exec": true,
   "actions": {
     "down": {
+      "comment": "Stop and destroy the docker containers",
       "command": "%action% %args%",
       "exec": false
     },
     "up": {
+      "comment": "Bring the docker containers up and live",
       "command": "%action% %args%",
       "exec": false
     },
     "pull": {
+      "comment": "Pull the latest versions of docker containers",
       "command": "%action% %args%",
       "exec": false
     },
     "start": {
+      "comment": "Start the docker containers",
       "command": "%action% %args%",
       "exec": false
     },
     "stop": {
+      "comment": "Stop the docker containers",
       "command": "%action% %args%",
       "exec": false
     },
-    "bash": "%action% %args%",
-    "composer": "%action% %args%",
+    "ip": {
+      "comment": "Print IP address of given container",
+      "service": "@host",
+      "command": "docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' %args%"
```

Comment:
in a proposal i would have asked:
- why do you need this? All containers have dns entries for the services. IPs aren't required.

Reply:
IPs arent normally required as internally they can reference each other using their names instead. however for example if i want to use gui client to connect to mysql container, then i need to use its IP as host.

Reply:
it can be removed if not so important. for me i use it frequently though

Reply:
I don't think you need it's IP, we always expose the port to host machine, I always enter localhost and the correct port and it works (the port of the host machine where it's exposed), but I do think it might be useful in some other use cases. But I can't think of one now. I'd prefer to have that option, it should be quite useful when required.